src/main/resources/com/google/api/codegen/java/sample.gradle.snip

```diff
@@ -1,185 +0,0 @@
-@snippet generate(metadata)
```

Comment:
Why is this file being deleted?

Reply:
It is not used anymore. I forgot to delete this file during one of my previous PRs.